r-package/grf/R/tune_regression_forest.R

```diff
@@ -155,7 +155,8 @@ tune_regression_forest <- function(X, Y,
                                num.fit.trees = tune.num.trees,
                                num.fit.reps = tune.num.reps,
                                num.optimize.reps = tune.num.draws,
-                               train = train)
+                               train = train,
+                               seed = seed)
```

Comment:
Instead of adding this argument you can access it from the `args` list (args$seed), same for the rest!

Reply:
I should have checked that!